nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -213,11 +219,22 @@
 
     public static final PropertyDescriptor PROP_SSL_CONTEXT_SERVICE = new PropertyDescriptor.Builder()
             .name("SSL Context Service")
-            .description("The SSL Context Service used to provide client certificate information for TLS/SSL (https) connections.")
+            .description("The SSL Context Service used to provide client certificate information for TLS/SSL (https) connections."
+                    + " It is also used to connect to HTTPS Proxy.")
             .required(false)
             .identifiesControllerService(SSLContextService.class)
             .build();
 
+    public static final PropertyDescriptor PROP_PROXY_TYPE = new PropertyDescriptor.Builder()
+            .name("Proxy Type")
+            .displayName("Proxy Type")
+            .description("The type of the proxy we are connecting to.")
+            .required(true)
+            .allowableValues(HTTP, HTTPS)
+            .defaultValue(HTTP.getValue())
+            .expressionLanguageSupported(true)
```

Comment:
Unfortunately, I believe it's not possible to use EL on properties with allowable values (see https://issues.apache.org/jira/browse/NIFI-1832). If we want to use EL on this property, I think it needs to be a classic text property with a custom validator a bit like in InvokeHTTP with the request method property (https://github.com/apache/nifi/blob/master/nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java#L152).

Reply:
thanks, I'll make it like this